src/Card/CardImageCap.jsx

```diff
@@ -47,24 +48,29 @@ const CardImageCap = React.forwardRef(({
 
   const handleSrcFallback = (event, altSrc) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
+      currentTarget.style.display = 'none';
+    } else if (currentTarget.src !== altSrc) {
       currentTarget.src = altSrc;
     }
   };
 
   return (
     <div className={classNames(className, wrapperClassName)} ref={ref}>
       <img
-        className="pgn__card-image-cap"
+        className={classNames('pgn__card-image-cap', { show: show.imageCap })}
         src={src}
         onError={(event) => handleSrcFallback(event, fallbackSrc)}
+        onLoad={() => setShow({ ...show, imageCap: src })}
```

Comment:
Using the `show` state variable in the `setShow` state setter function is an anti-pattern. Instead, to update state based on previous state, you can use the callback function approach to set state instead, i.e:

```jsx
setShow(s => ({ ...s, imageCap: true }))
```

Reply:
Yes, it`s true
Thanks